src/components/Button/src/Button.vue

```diff
@@ -132,10 +134,28 @@ function ghost(tokens) {
 	};
 }
 
+function transparent(tokens) {
+	const color = chroma(tokens.color);
+	const transparentHex = chroma('rgba(0, 0, 0, 0)').hex();
+	return {
+		'--small-padding': '0px',
+		'--medium-padding': '0px',
+		'--large-padding': '0px',
+		'--color-main': 'color',
+		'--color-main-hover': 'transparent',
+		'--color-main-active': 'transparent',
+		'--color-contrast': color.hex(),
+		'--color-contrast-hover': transparentHex,
+		'--color-contrast-active': transparentHex,
+		'--color-focus': transparentHex,
+	};
+}
+
 const VARIANTS = {
 	primary: fill,
 	secondary: outline,
 	tertiary: ghost,
+	text: transparent,
```

Comment:
not confident that `transparent` is the best name since `ghost` is also transparent, but open to suggestions